lib/seafoam/commands.rb

```diff
@@ -20,6 +20,7 @@ def seafoam(*args)
         @out.puts '        file.bgv[:graph][:node[-edge]] search term...'
         @out.puts '        file.bgv[:graph][:node[-edge]] edges'
         @out.puts '        file.bgv[:graph][:node[-edge]] props'
+        @out.puts '        file.bgv[:graph]:node source'
```

Comment:
Is the graph index really optional? `#source` looks like it requires it.

Reply:
Ah, not it's not - thanks!